changelog/README.md

```diff
@@ -1,11 +1,10 @@
-This directory contains "newsfragments" which are short files that contain a small 
+This directory contains "newsfragments" which are short files that contain a small
 **Markdown**-formatted text that will be added to the next `CHANGELOG`.
 
-The `CHANGELOG` will be read by **users**, so this description should be aimed 
-to Rasa OSS users instead of describing internal changes which are only relevant 
```

Comment:
I've removed reference to `internal changes` here as I thought it was misleading. We do add internal change log within misc as you mentioned. @ancalita